src/v/cloud_storage/cache_service.h

```diff
@@ -108,7 +108,8 @@ class cache : public ss::peering_sharded_service<cache> {
       config::binding<double>,
       config::binding<uint64_t>,
       config::binding<std::optional<double>>,
-      config::binding<uint32_t>) noexcept;
+      config::binding<uint32_t>,
+      config::binding<uint16_t>) noexcept;
```

Comment:
Would it be good to include parameter names here, or even a config structure? My PR adds 2 parameters to this, and the list is getting a little unwieldy.

Reply:
Will follow up when back from pto

Reply:
https://github.com/redpanda-data/redpanda/pull/22679